sdks/python/test-suites/portable/py36/build.gradle

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Instead of creating copies for each python versions, can we do something like https://github.com/apache/beam/blob/a6c9c92544643076c16d06e888c0152e39374564/runners/flink/1.8/job-server/build.gradle#L31